README.md

```diff
@@ -53,8 +53,17 @@ end
   - might not want to impement this
 - [ ] `Specter.close`  (ref, uuid)
 - [ ] RTC metrics sent to Elixir
-- [ ] `Specter.add_track`  (ref, uuid, ?)
-- [ ] `Specter.remove_track`  (ref, uuid, ?)
+- [ ] `Specter.rtcp_feedback/2` (string, string), returning `Specter.RTCPFeedback.t()`
```

Comment:
Maybe it would be better to have `Specter.RTCPFeedback.new`? The same for CodecCapability :thinking: